### DISCUSSION TITLE
Help with folder persmission

### DISCUSSION THREAD

**Discussion by mateusfg7** [requarks/wiki#7268] [Help / Questions] [answered]
There is a way to enable read/write access to a folder, and revoke to specific subfolder?
E.g., Assuming this folder structure:
```
root
├── Folder A (secret)
├── Folder B
├── Folder C
├── file1
└── file2		
```

There is a way to give to a group access to all folders and files on `root`, except `Folder A (secret)`?

**Comment by fnafox** [comment, marked as answer]
Create a DENY rule for that path for the group you don't want to have access to that folder. Deny rules override allow rules of the same level.
